File: 0062-unique-paths/0062-unique-paths.java
class Solution {
    public int uniquePaths(int m, int n) {
        int[][] map = new int[m + 1][n + 1];

        map[1][1] = 1;
        for (int i = 1; i < m + 1; i++) {
            for (int j = 1; j < n + 1; j++) {
                if (i == 1 && j == 1) continue;
                map[i][j] = map[i - 1][j] + map[i][j - 1];
            }
        }
        return map[m][n];
    }
}

File: 0062-unique-paths/0062-unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        r_map = [[1 for col in range(n)] for row in range(m)]

        for r in range(1, m):
            for c in range(1, n):
                r_map[r][c] = r_map[r-1][c] + r_map[r][c-1]

        return r_map[m-1][n-1]